src/index.js

```diff
@@ -5,16 +5,30 @@ import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter as Router } from "react-router-dom";
 import { ThemeProvider } from "./context/theme-context/theme-context";
+import {
+  AuthProvider,
+  CartProvider,
+  ToastProvider,
+  WishListProvider,
+} from "./context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <ThemeProvider>
-      <Router>
-        <App />
-      </Router>
+      <ToastProvider>
+        <AuthProvider>
+          <CartProvider>
+            <WishListProvider>
+              <Router>
```

Comment:
Can make a provider and add all these provider in it and keep index.js clean

Reply:
Will Check